WhisperAPI/WhisperAPI/Controllers/SuggestionsController.cs

```diff
@@ -39,5 +41,18 @@ public IActionResult GetSuggestions([FromBody] SearchQuery searchQuery)
 
             return this.Ok(suggestion);
         }
+
+        [HttpGet]
```

Comment:
HttpGet ? why not HttpPost like "public IActionResult GetSuggestions([FromBody] SearchQuery searchQuery)" ?

Reply:
why post?, we query previous data, we do not save or update anything new